LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationNetworkServerStartup.cs

```diff
@@ -124,6 +126,15 @@ public void ConfigureServices(IServiceCollection services)
 
             if (NetworkServerConfiguration.ClientCertificateMode is not ClientCertificateMode.NoCertificate)
                 _ = services.AddSingleton<IClientCertificateValidatorService, ClientCertificateValidatorService>();
+
+            _ = NetworkServerConfiguration.RunningAsIoTEdgeModule
+                ? services.AddSingleton<ModuleConnectionHost>()
```

Comment:
nit: should we harmonize the name ModuleControlHost ? (Could be done later)

Reply:
Will keep as it is for now 👍